app/validators/claim/advocate_claim_validator.rb

```diff
@@ -45,7 +45,7 @@ def self.fields_for_steps
 
   def validate_offence
     return if fixed_fee_case?
-    error_message = @record.agfs_reform? ? 'new_blank' : 'blank'
+    error_message = @record.agfs_reform? ? :new_blank : 'blank'
```

Comment:
should this not be `:blank` for it too pickup the error - 

```suggestion
    error_message = @record.agfs_reform? ? :new_blank : :blank
```